src/DotNetProjectFile.Analyzers/DotNetProjectFile.Analyzers.csproj

```diff
@@ -29,6 +29,7 @@
       <![CDATA[
 ToBeReleased
 - Proj0026: Remove IncludeAssets when redundant. (NEW RULE)
+- Proj0027: Update <TargetFrameworks> with <TargetFrameworks>. (NEW RULE)
```

Comment:
Maybe use the terminology override?

Reply:
I was in doubt. No'm sure. Lets go for override.